docs/coverage-reporter/index.md

```diff
@@ -289,7 +289,7 @@ Follow these instructions to validate that your coverage setup is working correc
             Coverage was uploaded for a commit that belongs to a branch that isn't analyzed by Codacy.
         </td>
         <td>
-            <p>Make sure that the <a href="../repositories-configure/managing-branches/">branch or target branch for pull requests is enabled on Codacy</a>.</p>
+            <p>Make sure that the <a href="../repositories-configure/managing-branches/">branch is enabled on Codacy</a>. Alternatively, open a pull request for the branch so that Codacy starts analyzing the branch automatically (as long as the target branch is enabled on Codacy).</p>
```

Comment:
@prcr
I tried to simplify even further. I'm unsure about mentioning that "codacy automatically starts analyzing"--can we reasonably expect that the user takes this outcome for granted?

Let me know if this still conveys the correct and complete meaning.

```suggestion
            <p>Make sure that the <a href="../repositories-configure/managing-branches/">branch is enabled on Codacy</a>. Alternatively, open a pull request and ensure that the target branch is enabled on Codacy.</p>
```

Reply:
> [...] can we reasonably expect that the user takes this outcome for granted?

I'm thinking that most users won't immediately make the association between opening a pull request and having the target branch enabled, and Codacy automatically enabling the analysis for the pull request branch. :thinking: 

We could still simplify a bit and avoid the parenthesis, while still including the consequence of Codacy automatically starting to analyze the branch:

```suggestion
            <p>Make sure that the <a href="../repositories-configure/managing-branches/">branch is enabled on Codacy</a>. Alternatively, ensure that the target branch is enabled and open a pull request for Codacy to start analyzing the branch automatically.</p>
```

Reply:
I like this one!